README.md

```diff
@@ -1 +1,34 @@
-# Purdue ECEA Website
\ No newline at end of file
+# Purdue ECEA Website
+
+[![Build Status](https://travis-ci.com/Purdue-ECESS/ecea-website-source-code.svg?branch=main)](https://travis-ci.com/Purdue-ECESS/ecea-website-source-code)
+
+## Installation Guide
+
+### Installing Node
+Go to https://nodejs.org/en/download. Instructions for Linux, Windows, and Mac should all be on there. 
+
+
+### Installing Git
+Windows: https://git-scm.com/download/win
+Mac: https://git-scm.com/download/mac
+Linux: https://git-scm.com/download/linux
+
+
+### Install Purdue ECEA Source Code
+Type these commands inside your window / bash terminal. Installing git allows you to deploy your changes to the website. 
+
+git clone https://github.com/Purdue-ECESS/ecea
+cd ecea
+
+
+### Installing Next + Required Tools + Running Application
+npm install
+npm install --save-dev typescript @types/react 
```

Comment:
I don't think you need this step

Reply:
For me I need the 'npm install', otherwise it won't build, at least for the first time. I took out the 'npm install --save-dev typescript @types/react' though